manifests/m/Microsoft/PCManager/Beta/3.9.4.0/Microsoft.PCManager.Beta.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.PCManager.Beta
+PackageVersion: 3.9.4.0
+InstallerType: exe
+InstallModes:
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+UpgradeBehavior: install
+ReleaseDate: 2023-12-04
+Installers:
+- Architecture: x64
+  InstallerUrl: https://files03.tchspt.com/down/MSPCManagerSetup.exe

```

Comment:
This doesn't appear to match the publisher, and this application appaers to not be released yet. Can you help me to find a link to this package on the publisher's site?

Reply:
https://www.techspot.com/downloads/7506-microsoft-pc-manager.html

Reply:
Exactly - that doesn't appear to be a Microsoft-owned domain. 
> © 1998 - 2024 TechSpot, Inc. All Rights Reserved.

https://www.techspot.com/about/
> TechSpot is a registered trademark and the website techspot.com is property of TechSpot, Inc.

This appears to be the domain of TechSpot, a different company.